hbase-thrift/src/main/resources/org/apache/hadoop/hbase/thrift/Hbase.thrift

```diff
@@ -192,6 +192,14 @@ exception AlreadyExists {
   1:string message
 }
 
+/**
+ * Specify type of thrift server: thrift and thrift2
+ */
+enum TThriftServerType {
```

Comment:
rather than defining the same enum twice in both the thrift files, is it better to have a const defined? 

const i16 THRIFT_SERVER_TYPE = 1; // in thrift1
const i16 THRIFT_SERVER_TYPE =2 // in thrift2

Each thrift server impl can use it's corresponding type directly. Thoughts?

Reply:
I think it's better to use TThriftServerType for readability